src/app/components/ParticipatedContests/config.js

```diff
@@ -0,0 +1,6 @@
+export const TB_CONFIG = {
+  colWidths: [null, 540, 123, 121],
```

Comment:
we should have fixed width for `index`, and max width for `contestName`. So it should be `[x, null, 123, 121]` with `x` is the fixed width of index col